### ISSUE TITLE
PNGQuant - Unrecognized option Error

### ISSUE THREAD

**Issue by patricklindsay** [jtescher/image_optimizer#17]
I'm receiving the following error when trying to compress an image using `PNGQuant` (v2.10.2).

```Unrecognized option: 'speed 1'```

From investigation it looks like there is a problem with splatting the `command_options`.

Below is my current workaround; 
```
ImageOptimizer::PNGQuantOptimizer.class_eval do
   private

   def perform_optimizations
     system(command_options.join(' '))
   end

   def command_options
     flags = ['pngquant', '--skip-if-larger', '--force', '--verbose', '--ext .png']
     flags << path
   end
end
```

**Comment by jtescher** [comment]
Hm looks good. Want to do a PR for this?

**Comment by jtescher** [comment]
Fixed via #18
